.babelrc

```diff
@@ -0,0 +1,6 @@
+{
+  "presets": [
```

Comment:
true, I had to add it because of the tests but I will move it into the tests file as well and get rid of .babelrc (see how it looks like)